openelevationservice/server/api/querybuilder.py

```diff
@@ -49,6 +54,7 @@ def format_PixelAsGeoms(result_pixels):
            func.unnest(literal_column("ARRAY{}".format(heights)))
 
 
+###Start-Original code for the polygon_coloring_elevation function
```

Comment:
It is not necessary to comment on where a function starts and where it ends.

Reply:
Done. Deleted.